src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -203,4 +203,36 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
         }
       }
     }
+
+  @Path("/{version}/{workflowId}/outputs/{callFqn}")
+  @ApiOperation(
+    value = "Query for call outputs based on workflow id and call fully qualified name (e.g. my_workflow.my_call).",
+    nickname = "call-outputs",
+    httpMethod = "GET",
+    produces = "application/json"
+    //,response = classOf[CallOutputs]
```

Comment:
I believe this was commented out of every annotation like this a while ago because we couldn't get something working (Swagger related?) but other uses of this annotation in other projects used the `response =`, so we left it in as a reminder or something... Probably just cruft at this point though.